pkg/local_object_storage/shard/delete.go

```diff
@@ -57,14 +57,6 @@ func (s *Shard) deleteObjs(addrs []oid.Address) error {
 
 	var totalRemovedPayload uint64
 
-	for i := range res.RemovedObjects {
```

Comment:
Makes the behavior different in case of `IsErrNotFound`, but at the same time I'm not sure what is the correct one here. The object is gone, but was it deleted from counters previously (or the file has evaporated), should it be deleted now (potentially twice)?

Reply:
noticed this too. Will just revert previous behavior. Other thoughts are outside this issue